### PR TITLE
added toast to OrderSummary component and fixed the return url after …

### PR DESCRIPTION
**Pull request by joybiswas704**
…payment process

### CODE REVIEW

File: src/components/Checkout/OrderSummary.js
Comment (saifulislamdev) [inline src/components/Checkout/OrderSummary.js:16]: Are you sure we need this? You weren't able to use `useHistory` or `useLocation` to do this?

File: src/components/Checkout/OrderSummary.js
Comment (saifulislamdev) [inline src/components/Checkout/OrderSummary.js:41]: Do you want to send it to `/checkout` as you had before?

File: src/components/Checkout/OrderSummary.js
Comment (joybiswas704) [reply src/components/Checkout/OrderSummary.js:16]: I actually just learned about it and yes i might use that

File: src/components/Checkout/OrderSummary.js
Comment (joybiswas704) [reply src/components/Checkout/OrderSummary.js:41]: I initially had checkout since i didnt have any other pages. Eventually i will use order page for the return url. For now instead of sending the user to the same checkout page im sending them to cart page

File: src/util/config.js
Comment (saifulislamdev) [inline src/util/config.js:10]: Maybe use the `window` object instead of creating another env variable? I don't think it should be a problem with Netlify.

**Review by saifulislamdev — CHANGES_REQUESTED** [review]
Left some comments. Let's merge this PR after the previous PRs have been merged.

**Pull request closed without merging**
